lib/lookup.json

```diff
@@ -507,6 +507,9 @@
     "skip": "win32",
     "maintainers": "jashkenas"
   },
+  "undici": {
+    "maintainers": ["mcollina", "ronag"]
```

Comment:
Seems like it requires the prefix `v`, or am I mistaken?

Reply:
how could I now? how is the prefix picked up? Is it in the github tag?

Reply:
It is usually not necessary to specify the tag prefix. If the npm package is published from a git repo, npm sets the git SHA in the package metadata and we use that to download the tarball.